lib/ui/login/login_screen.dart

```diff
@@ -0,0 +1,84 @@
+  const LoginScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    SystemChrome.setSystemUIOverlayStyle(const SystemUiOverlayStyle(
+        statusBarColor: Colors.transparent,
+        statusBarIconBrightness: Brightness.light,
+        statusBarBrightness: Brightness.dark));
+    return Scaffold(
+        backgroundColor: Colors.transparent,
+        body: Container(
+          decoration: BoxDecoration(
+            image: DecorationImage(
+              image: AssetImage(Assets.images.bgLoginOverlay.path),
+              fit: BoxFit.cover,
+            ),
+          ),
+          child: Padding(
+            padding:
+                const EdgeInsets.symmetric(horizontal: Dimensions.paddingLarge),
+            child: Center(
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  Padding(
+                      padding: const EdgeInsets.only(
+                          bottom: 109.0, left: 104.0, right: 104.0),
+                      child: Image.asset(
+                        Assets.images.icNimbleWhiteLogo.path,
+                      )),
+                  Padding(
+                      padding: const EdgeInsets.only(
+                          bottom: Dimensions.paddingMedium),
+                      child: InputFieldWidget(
+                        key: const Key('login_email'),
+                        textHint: AppLocalizations.of(context)!.loginEmail,
+                        keyboardType: TextInputType.emailAddress,
+                        textInputAction: TextInputAction.next,
+                      )),
+                  Padding(
+                      padding: const EdgeInsets.only(
+                          bottom: Dimensions.paddingMedium),
+                      child: InputFieldWidget(
+                          key: const Key('login_password'),
+                          textHint: AppLocalizations.of(context)!.loginPassword,
+                          keyboardType: TextInputType.visiblePassword,
+                          isPassword: true,
+                          textSuffixButton:
+                              AppLocalizations.of(context)!.loginForgotPassword,
+                          textSuffixButtonCallback: null)),
+                  ElevatedButton(
+                    onPressed: () {},
+                    style: ElevatedButton.styleFrom(
+                      minimumSize:
+                          const Size.fromHeight(Dimensions.buttonHeight),
+                      backgroundColor: Colors.white,
+                      shape: RoundedRectangleBorder(
+                        borderRadius:
+                            BorderRadius.circular(Dimensions.radiusNormal),
+                      ),
+                    ),
+                    child: Text(
+                      AppLocalizations.of(context)!.login,
+                      style: Theme.of(context)
+                          .textTheme
```

Comment:
Please use `context.textTheme` instead.

```suggestion
                      style: context.textTheme
```

Reply:
Same with `textTheme` 🤔 

<img width="1000" alt="Screenshot 2023-04-12 at 2 13 35 PM" src="https://user-images.githubusercontent.com/25218255/231380372-50459f48-3246-4c84-9f15-681e2bd1f3d3.png">

Reply:
About the `context.textTheme` I mentioned in [the closed PR](https://github.com/nimblehq/flutter-ic-david-bruce/pull/42). I think you missed the context of my comments. 😄 

No worries, I put the preferences [here](https://github.com/nimblehq/flutter-ic-david-bruce/pull/42#discussion_r1160536854)